charts/agent-control/templates/uninstall.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if (index .Values "agent-control-deployment").enabled -}}
+{{- $uninstallJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "uninstall-job" ) -}}
+{{- $secretName := include "agent-control.secret.name" . -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    helm.sh/hook-weight: "-10"
+  name: {{ $uninstallJobName }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  backoffLimit: 2
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" .Release.Name "suffix" "uninstall-job") }}
+      containers:
+        - name: uninstall-agent-control-deployment
+          image: newrelic/newrelic-agent-control-cli:nightly
```

Comment:
why is it hardcoded? I think they are templated in the values.yaml

Reply:
Yes. It was hardcoded by @rubenruizdegauna and I didn't change it. Sorry.